templates/deno/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "importMap": ".vscode/resolve_npm_imports.json",
+  "compilerOptions": {
+    "lib": ["dom", "dom.iterable", "dom.asynciterable", "deno.ns"]
+  }
+}
```

Comment:
[Deno auto-detects `deno.json`](https://deno.land/manual/getting_started/configuration_file#configuration-file), so let's use that instead of `tsconfig.json`